src/usbasp.c

```diff
@@ -586,24 +567,21 @@ static int usbasp_open(PROGRAMMER *pgm, const char *port) {
       if (usbOpenDevice(&PDATA(pgm)->usbhandle, USBASP_OLD_VID, "www.fischl.de",
 		             USBASP_OLD_PID, "USBasp") == 0) {
         /* found USBasp with old IDs */
-        avrdude_message(MSG_INFO, "%s: Warning: Found USB device \"USBasp\" with "
-                        "old VID/PID! Please update firmware of USBasp!\n",
-                        progname);
+        pmsg_error("found USB device USBasp with old VID/PID; please update firmware of USBasp\n");
```

Comment:
This sentence sounds a bit odd

Reply:
> This sentence sounds a bit odd

Do you have a suggestion for a replacement?

Reply:
For instance: "found USBasp programer with old[/outdated?] VID/PID; please update [the] USBasp firmware\n"

Reply:
Cool